products/sle/main.pm

```diff
@@ -1027,6 +1027,39 @@ sub load_slenkins_tests {
     return 0;
 }
 
+sub load_hpc_cluster_tests {
+    # We need to boot *before* waiting for support server
+    # Because of TianoCore UEFI boot method
+    loadtest "boot/boot_to_desktop";
+
+    # Wait for support server to complete its initialization
+    # Inspired by HA test, but we can't reuse it as it is
+    loadtest "hpc/wait_support_server";
```

Comment:
recently I add this https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/support_server/wait_support_server.pm , you might use it in your PR

Reply:
I see, but it has line ```barrier_wait('BARRIER_HA_' . get_var('CLUSTER_NAME')) if get_var("HA_CLUSTER");``` for which i decide not to use it. Line was removed from hpc version of this file. Although it can work, because barrier is not initialized, but it doesn't look clean. I suppose that in generic support_server files, shouldn't be any specifics related to particular tests.

Reply:
You are right, we can reuse it.